src/explore-education-statistics-frontend/src/hooks/useCookies.ts

```diff
@@ -0,0 +1,57 @@
+import { useCookies as useBaseCookies } from 'react-cookie';
+import {
+  enableGA,
+  disableGA,
+  googleAnalyticsCookies,
+} from '@frontend/services/googleAnalyticsService';
+import { addMonths } from 'date-fns';
+
+interface Cookie {
+  name: string;
+  expires: Date;
+  duration?: string;
+}
+
+interface CookieMap {
+  bannerSeen: Cookie;
+  disableGA: Cookie;
+}
+
+export const cookieMap: CookieMap = {
+  bannerSeen: {
+    name: 'ees_banner_seen',
+    expires: addMonths(new Date(), 1),
+    duration: '1 month',
+  },
+  disableGA: {
+    name: 'ees_disable_google_analytics',
+    expires: addMonths(new Date(), 1),
```

Comment:
If they disable this, shouldn't this never expire?

Reply:
I think this is fine, as Google analytics will only initialise once cookies are accepted or if cookie settings are saved with GA turned on.

When the banner reappears after a month, and the user selects `Accept Cookies`, then I would expect google analytics to be turned on at this point, however currently this will only happen if the googleAnalytics isDisabled cookie isn't set to true..